src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -1319,6 +1319,76 @@ def _validate_liquid_separation_efficiency(self, attribute, value):
         assert isinstance(value, Scalar) and 0.6 <= value.GetValue("-") <= 1.0
 
 
+@attr.s(slots=True, kw_only=True)
+class ControllerInputSignalPropertiesDescription:
```

Comment:
Hey, nice work!

As it still fresh on your mind, could you please add some docs here?

Reply:
Sure